src/js/stores/MarathonStore.js

```diff
@@ -160,6 +160,12 @@ var MarathonStore = Store.createStore({
   },
 
   processMarathonGroups: function (data) {
+    data.filterProperties = {
+      name: function (item) {
+        return item.id;
+      }
+    };
+
```

Comment:
I think this mutation does not belong here. I need to think about this a bit more.